packages/builder/src/utils/graphql.ts

```diff
@@ -17,8 +8,7 @@ export enum ChainId {
  */
 const getGraphQLEndpoint = (chainId: number): string | undefined => {
```

Comment:
we need to add the subgraph for PGN

Reply:
Yes - and I will change this object to be typesafe once we have that URL - meaning it will fail typecheck if the subgraph URL isn't specified for all chains. (and I will also check that it's not an empty string so that we don't "cheat" it by just putting an empty string hehe)